[CARBONDATA-4010] Doc changes for long strings. (#3959)
apache/carbondata: docs/ddl-of-carbondata.md

   - ##### String longer than 32000 characters

     In common scenarios, the length of string is less than 32000,
     so carbondata stores the length of content using Short to reduce memory and space consumption, and it handles the strings

>      which length greater than 32000 as bad record. Refer [bad record handling](https://github.com/apache/carbondata/blob/master/docs/dml-of-carbondata.md#bad-records-handling) section for better understanding.

Reviewer: ```suggestion
 and it handles strings which have length greater than 32000 as a bad record.

```

Author: done

       ```
       ALTER TABLE tablename UNSET TBLPROPERTIES('SORT_SCOPE')
       ```
     - ##### Long String Columns
       Example to SET Long String Columns:
       ```
       ALTER TABLE tablename SET TBLPROPERTIES('LONG_STRING_COLUMNS'='column1')
       ```
       **NOTE:** Only string columns can be set to long string columns. Cannot set sort columns to long string columns.

       Example to UNSET Long String Columns:
       ```
       ALTER TABLE tablename UNSET TBLPROPERTIES('LONG_STRING_COLUMNS')
       ```

>        **NOTE:** On unset long string columns are set to their original datatypes.

Reviewer: ```suggestion
       **NOTE:** On unset, long string columns are set to their original datatypes.
```

Author: done
